index.html

```diff
@@ -2554,42 +2556,52 @@ <h3>
             The user agent is free to display the icon in any context.
           </dd>
         </dl>
+        <p class="note">
+          If an icon contains multiple purposes, it can be used for any of
```

Comment:
Maybe: 
> used by the user agent for any of

Otherwise, it's a little unclear if the note is speaking to developers.